dev-util/devscripts/devscripts-2.16.4.ebuild

```diff
@@ -0,0 +1,80 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_{3,4,5} )
+DISTUTILS_OPTIONAL=true
+
+inherit distutils-r1
+
+DESCRIPTION="Scripts to make the life of a Debian Package maintainer easier"
+HOMEPAGE="https://anonscm.debian.org/gitweb/?p=collab-maint/devscripts.git"
+SRC_URI="mirror://debian/pool/main/d/${PN}/${PN}_${PV}.tar.xz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="
+	dev-lang/perl
+	python? ( ${PYTHON_DEPS} )
+"
+
+RDEPEND="${DEPEND}
+	app-arch/dpkg
+	dev-util/debhelper
+	sys-apps/fakeroot"
+
+IUSE="python"
+
+REQUIRED_USE="python? ( ${PYTHON_REQUIRED_USE} )"
+
+src_prepare() {
+	default
+
+	if use python; then
+		cd "${S}"/scripts || die
```

Comment:
Conditionally changing directory can be a bit dangerous when someone else modifies the ebuild, and forgets to use full path below. I suggest you `cd ../.. || die` afterwards, or `cd - >/dev/null || die` (which cds to previous directory), or pushd/popd.